pyproject.toml

```
@@ -9,18 +9,22 @@ author-email = "blaise@untzag.com"
 home-page = "https://github.com/wright-group/PyCMDS"
 classifiers = ["License :: OSI Approved :: MIT License"]
 requires-python = ">=3.7"
-requires = ["click",
+requires = ["appdirs",
+            "click",
             "numexpr",
             "psutil",
             "pydaqmx",
  	    "pydrive",
 	    "pyqtgraph",
+	    "pyside2",
 	    "pyserial",
 	    "slackclient",
```

```suggestion
	    "slackclient<2",
```

We are currently using an old version of slackclient, and are not compatible with the most recent version